noir-projects/noir-protocol-circuits/crates/types/src/abis/partial_state_reference.nr

```diff
@@ -0,0 +1,44 @@
+use crate::{
```

Comment:
This is just moved from the parent dir and the manual implementations of serde were nuked from here.